components/org.wso2.carbon.identity.scim.provider/src/main/java/org/wso2/carbon/identity/scim/provider/util/SCIMProviderConstants.java

```diff
@@ -52,6 +52,8 @@
     public static final String DISPLAY_NAME = "displayName";
     public static final String WILDCARD_ASTERISK = "*";
 
+    public static final String ELEMENT_NAME_SHOW_ALL_USER_DETAILS = "ShowAllUserDetails";
```

Comment:
Seems like we have to change the identity.xml tag name from "ShowAllUserDetails" to something else because name it self-mentioning that we are going to show all the user details but actually we are going to show only one user details. Shall we use a more appropriate name?